src/components/JobOffers/JobOffersList/JobOfferCard/JobOfferCard.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import './JobOfferCard.scss'
+
+export class JobOffer {
+    id: string;
+    title: string;
+    companyName: string;

```

Comment:
**nitpick:** Let's refactor class to be an interface. You don't need class here, as it increases complexity - you need just:

```ts
export class JobOfferProps {
    id: string;
    title: string;
    companyName: string;
    city: string;
    country: string;
    workLocation: string;
    seniority: string;
    salaryFrom: number;
    salaryTo: number;
    currency: string;
    image: string;
    createdAt: string;
}